### PR TITLE
Release/r7

### PR DESCRIPTION
**Pull request by livingrockrises**
# Summary

Related Issue: # (issue number)

## Change Type

- [ ] Bug Fix
- [ ] Refactor
- [ ] New Feature
- [ ] Breaking Change
- [ ] Documentation Update
- [ ] Performance Improvement
- [ ] Other

# Checklist

- [ ] My code follows this project's style guidelines
- [ ] I've reviewed my own code
- [ ] I've added comments for any hard-to-understand areas
- [ ] I've updated the documentation if necessary
- [ ] My changes generate no new warnings
- [ ] I've added tests that prove my fix is effective or my feature works
- [ ] All unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published

# Additional Information

# Branch Naming

### CODE REVIEW

**Review by AmanRaj1608 — APPROVED** [review]
LGTM!

**Review by VGabriel45 — APPROVED** [review]

File: packages/account/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,13 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 3.1.2 (2023-12-28)
+
+
+### Bug Fixes
+
+* use undefined in place of ! + check on limits returned by paymaster and throw ([0376901](https://github.com/bcnmy/biconomy-client-sdk/commit/0376901b7aec8c268a6a3c654d147335974d78f3))
+
```
Comment (NunoFilipeSantos) [inline packages/account/CHANGELOG.md:12]: We're missing some:

https://github.com/bcnmy/biconomy-client-sdk/pull/342
https://github.com/bcnmy/biconomy-client-sdk/pull/340
https://github.com/bcnmy/biconomy-client-sdk/pull/337
https://github.com/bcnmy/biconomy-client-sdk/pull/336
https://github.com/bcnmy/biconomy-client-sdk/pull/335
https://github.com/bcnmy/biconomy-client-sdk/pull/332

Comment (livingrockrises) [reply packages/account/CHANGELOG.md:12]: 340 is not going into this release r7

Comment (livingrockrises) [reply packages/account/CHANGELOG.md:12]: 335 is already covered in this

Comment (livingrockrises) [reply packages/account/CHANGELOG.md:12]: updated for the rest

Comment (livingrockrises) [reply packages/account/CHANGELOG.md]: https://github.com/bcnmy/biconomy-client-sdk/commit/1868d71158f79c84cf97d09fdbbf22b455eecb7c

**Pull request merged**